build-new-version.sh

```diff
@@ -105,6 +105,34 @@ download_files() {
     done
 }
 
+cargo_apache_expected="1836efb2eb779966696f473ee8540542"
+cargo_mit_expected="b377b220f43d747efdec40d69fcaa69d"
+
+check_cargo_license() {
```

Comment:
Looks like there might be a mix of tabs and spaces in this function, things are aligned weird.

Reply:
Well that's embarrassing. Fixed below.